src/codegen.cpp

```diff
@@ -1293,15 +1299,24 @@ void jl_extern_c(jl_function_t *f, jl_value_t *rt, jl_value_t *argt, char *name)
     assert(jl_is_tuple_type(argt));
     Function *llvmf = jl_cfunction_object(f, rt, (jl_tupletype_t*)argt);
     if (llvmf) {
-        #ifndef LLVM35
-        new GlobalAlias(llvmf->getType(), GlobalValue::ExternalLinkage, name, llvmf, llvmf->getParent());
-        #elif defined(LLVM37) && !defined(LLVM38)
-        GlobalAlias::create(cast<PointerType>(llvmf->getType()),
-                            GlobalValue::ExternalLinkage, name, llvmf, llvmf->getParent());
-        #else
-        GlobalAlias::create(llvmf->getType()->getElementType(), llvmf->getType()->getAddressSpace(),
-                            GlobalValue::ExternalLinkage, name, llvmf, llvmf->getParent());
-        #endif
+        Function *active_llvmf = active_module->getFunction(llvmf->getName());
+        // In imaging mode, and in most cases in JIT mode (where the wrapper is specifically)
+        // compiled for this function, we can simply use a global alias.
```

Comment:
is this supposed to be "(where the wrapper is specifically compiled for this function)" ? the current parenthetical doesn't make much sense